### ISSUE TITLE
Autocomplete: input, label & trigger are missing from the build

### ISSUE THREAD

**Issue by ilianiv** [qwikifiers/qwik-ui#396]
Although I've seen implementations for them (https://github.com/qwikifiers/qwik-ui/tree/main/packages/kit-headless/src/components/autocomplete), they aren't present in the build.
![image](https://github.com/qwikifiers/qwik-ui/assets/3925630/3ff7a823-48bb-4bca-9a26-ec944445f316)
![image](https://github.com/qwikifiers/qwik-ui/assets/3925630/278328bf-7353-4634-87de-565d402c2fcf)

**Comment by shairez** [comment]
Yeah I think we might have pushed a newer version of the docs without releasing a new version

@thejackshelton can you verify if this is the case and if we can release a new version from the current `main` branch?

**Comment by thejackshelton** [comment]
@shairez I don't see anything wrong in v0.1.13 that would cause this issue, but in my stackblitz example I get the same problem.

Not getting that in my local fork on the website examples. Maybe releasing a new version might fix the issue in the process?

**Comment by thejackshelton** [comment]
It could be that we're exporting the root twice I believe:

```tsx
export * from './components/autocomplete/autocomplete-root';
export * from './components/autocomplete/';
```

**Comment by gparlakov** [comment]
It would seem that Qwik needs to export some more of its types 

Details here https://github.com/BuilderIO/qwik/issues/5140 - issue with minimal repro and a PR with a minimal fix :)
